espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java

```diff
@@ -40,8 +40,11 @@
     @Test
     public void startEspressoServer() throws InterruptedException, IOException, DuplicateRouteException {
         new Server();
-        // TODO: Figure out how to keep Runner open forever
-        Thread.sleep(3000000);
         assertEquals(true, true); // Keep Codacy happy
+
+        // The server should run permanently. The test is only closed
+        while (true) {
```

Comment:
while true is not needed here. Long.MAX_VALUE is more that thousand years ;)

Reply:
@mykola-mokhnach Gotta cover the edge cases.

Reply:
Pushed a fix.